tests/task/test_calcZernikesDanishTaskCwfs.py

```diff
@@ -56,33 +56,41 @@ def setUpClass(cls):
         cls.testDataDir = os.path.join(moduleDir, "tests", "testData")
         testPipelineConfigDir = os.path.join(cls.testDataDir, "pipelineConfigs")
         cls.repoDir = os.path.join(cls.testDataDir, "gen3TestRepo")
-        cls.runName = "run1"
+        cls.cameraName = "LSSTCam"
```

Comment:
This gets set also in the previous version of the test, but since it's not called anywhere, I wonder if it's needed at all.